### DISCUSSION TITLE
Extended Keyboard shortcuts for indent / outdent on iOS

### DISCUSSION THREAD

**Discussion by malsharman** [vincode-io/Zavala#109] [Ideas]
Would love to see extended keyboard buttons for adding indents/outdent. To me that's a lot easier than the current drag to indent/outdent support. Its a pretty common feature for outliners on iOS to do it with buttons

**Comment by vincode-io** [comment]
I'm open to it if people think it is worth losing the vertical space.

**Reply by vincode-io** [reply]
I've added this and it will be in the upcoming 1.1 release.
